app/assets/javascripts/components/course_creator/course_creator.jsx

```diff
@@ -413,115 +434,151 @@ const CourseCreator = createReactClass({
       );
     }
 
+    const programs = [
+      {
+        name: 'Generic course',
+        type: 'BasicCourse',
+        description: 'Pastry danish fruitcake liquorice gingerbread. Liquorice pie wafer sesame snaps chupa chups sweet roll. Toffee marzipan sugar plum caramels.'
```

Comment:
This program type will track edits to any articles or mainspace pages by any participants. If automated wiki edits are enabled on the program's home wiki, then an on-wiki version of this program will be automatically created and updated.